provider/aws/config.go

```diff
@@ -84,8 +83,10 @@ func newV2Config(awsConfig AWSSessionConfig) (awsv2.Config, error) {
 		config.WithRetryer(func() awsv2.Retryer {
 			return retry.AddWithMaxAttempts(retry.NewStandard(), awsConfig.APIRetries)
 		}),
-		config.WithHTTPClient(extdnshttp.NewInstrumentedClient(&http.Client{})),
+		config.WithHTTPClient(awshttp.NewBuildableClient()),
+
 		config.WithSharedConfigProfile(awsConfig.Profile),
+		config.WithAPIOptions(GetInstrumentationMiddlewares()),
```

Comment:
Any chance to share that the metric is still works. It should capture calls to AWS endpionts, example screenshot is here https://github.com/kubernetes-sigs/external-dns/pull/5650#issue-3231826660